### PR TITLE
Added Skittles to homepage, fixed inconsistencies based on Kaits feedback

### PR DESCRIPTION
**Pull request by stowellm**
Addresses #1659

### CODE REVIEW

**Comment by stowellm** [comment]
My Travis build is failing due to:

```
Fetching https://github.com/asalani93/better-colorpicker
fatal: Could not parse object '6e05792fdd4415d9bcc66c627650dc9da1b95adb'.
Git error: command `git reset --hard 6e05792fdd4415d9bcc66c627650dc9da1b95adb`
in directory
/home/travis/.rvm/gems/ruby-2.1.1/bundler/gems/better-colorpicker-6e05792fdd44
has failed.
If this error persists you could try removing the cache directory
'/home/travis/.rvm/gems/ruby-2.1.1/cache/bundler/git/better-colorpicker-ad84e9c82d84d9c832c465a5d46aa4eec3268e0d'
```

@asalani93, do you know what could be causing this?

**Comment by stowellm** [comment]
Still fails even when switching to `isenseDev/better-colorpicker`.  Sorry, not sure what this is about.

**Comment by asalani93** [comment]
better-datetimepicker's version is still 0.0.4, which isn't the current one on the repo.  However, that shouldn't be causing better-colorpicker to fail.

I'm using: 
```
gem 'better_datetimepicker', '~> 0.0.5', git: 'https://github.com/isenseDev/better-datetimepicker'
gem 'better_colorpicker', '~> 0.0.3', git: 'https://github.com/isenseDev/better-colorpicker'
```
... and that seems to work fine, both locally and on Travis.

**Comment by stowellm** [comment]
Thanks @asalani93 - looks like its working now.  Ready for review once (or should I say *if* :angry:) it passes Travis.

File: db/schema.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20141010131030) do
+ActiveRecord::Schema.define(version: 20141015152508) do
```
Comment (stowellm) [inline db/schema.rb]: Someone mind explaining to me what this file is for, and why now and again it auto-updates and gets included in a pull request?  I've seen some people even roll back the file.

**Comment by kcarcia** [comment]
@stowellm Sorry about the trouble here. I pulled the latest version so Anthony's PR could get through testing. I thought I'd review and merge his PR pretty shortly after, but I didn't come in today since campus was closed.

I will locally pull and review this tomorrow morning.

File: db/schema.rb (hunk 1 above)
Comment (jaypoulz) [inline db/schema.rb]: The schema file defines the current state of the data base (PostgreSQL or SQLite). It auto-updates every time a new migration is written and run on the data base. It gets reverted when we roll back the data base.

**Pull request merged**